docs/how-to/rocm-for-ai/model-automation-and-dashboarding.rst

```diff
@@ -0,0 +1,154 @@
+.. meta::
+   :description: Discover, run, and benchmark deep learning models with AMD MAD -- Model Automation and Dashboarding tool.
+   :keywords: dashboard, machine, deep, container, playground, tune
+
+************************
+Running models using MAD
+************************
+
+The AMD Model Automation and Dashboarding (MAD) tool integrates an AI model zoo with automated execution capabilities
+across various GPU architectures. It facilitates performance tracking by maintaining historical performance data and
+generating dashboards for analysis. MAD's source code repository and full documentation can be found at
+`<https://github.com/ROCm/MAD>`__.
+
+MAD retrieves various models from their repositories and tests their performance in ROCm Docker images. It is an index
+of deep learning models optimized for reproducible accuracy and performance on AMD GPUs and accelerators using the ROCm
+software stack.
+
+Use MAD to:
+
+*  Try new models,
+
+*  Compare performance between patches or architectures, and
+
+*  Track functionality and performance over time.
+
+Getting started with MAD
+========================
+
+Refer to the procedures in :doc:`rocm-install-on-linux:index` to set up your host computer with ROCm. Follow the
+detailed :doc:`installation instructions <rocm-install-on-linux:install/detailed-install>` for Linux-based platforms.
+
+ROCm Docker images
+------------------
+
+You can find ROCm Docker images for PyTorch and TensorFlow on Docker Hub at
+`rocm/pytorch <https://hub.docker.com/r/rocm/pytorch>`_ and
+`rocm/tensorflow <https://hub.docker.com/r/rocm/tensorflow>`_.
+
+AMD publishes a unified Docker image at `rocm/vllm <https://hub.docker.com/r/rocm/vllm>`_ that packages
+together vLLM and PyTorch for the AMD Instinct™ MI300X accelerator. This enables users to quickly validate the expected
+inference performance numbers on the MI300X. This Docker image includes:
```

Comment:
```suggestion
AMD publishes a unified Docker image at `rocm/vllm <https://hub.docker.com/r/rocm/vllm>`_ which packages vLLM and PyTorch for the AMD Instinct™ MI300X accelerator. This enables youo quickly validate the expected
inference performance numbers on the MI300X. This Docker image includes:
```

Reply:
I'm usually careful around phrases like:
AMD publishes...

Perhaps instead:
An AMD unified docker is available... or soemthing